### ISSUE TITLE
depositEthAsync is running but isn't translating to a plasmachain balance

### ISSUE THREAD

**Issue by michaelcohen716** [alice-finance/alice.js#1]
I'm testing the transfer gateway on rinkeby using ```alice.getEthereumChain().depositEthAsync(value)``` . The events log and I see the tx on etherscan but I'm not seeing my Loom net eth balance increase using ```alice.getLoomChain().balanceOfETHAsync()``` . I checked again an hour after running the transaction and there still is a zero balance, so this issue is not related to the block confirmation delay. Where is the issue here?

**Comment by YoonjaeYoo** [comment]
You need to wait for 10 block confirmation in ethereum for the transfer gateway oracle to start the deposit process.

**Comment by michaelcohen716** [comment]
I mentioned in my first post that I checked again **an hour** after I initially deposited, more than enough for 10 block confirmations. So this doesn't explain my issue

**Comment by YoonjaeYoo** [comment]
Did you map accounts using code below?
```
const mapped = await alice.areAccountsMapped();
if (!mapped) {
    await alice.mapAccounts();
}
```

**Comment by michaelcohen716** [comment]
yes i've been following the docs exactly. 

```alice.getEthereumChain().getETHReceivedLogsAsync()``` is returning all my attempts at transfers correctly. the issue is that ```alice.getLoomChain().balanceOfETHAsync()``` continues to return zero

**Comment by YoonjaeYoo** [comment]
If you're on loom mainnet, you can check your account's ETH balance below.

http://plasma-blockexplorer.dappchains.com/

Do you see any balance?

**Comment by michaelcohen716** [comment]
I'm testing on rinkeby + extdev to get it working first, so won't have a mainnet balance. Is there an example of working code for depositing eth with alice beyond the readme docs?

**Comment by YoonjaeYoo** [comment]
http://extdev-blockexplorer.dappchains.com/

Above is the block explorer for extdev. You can check your balance.

**Comment by michaelcohen716** [comment]
nothing

**Comment by YoonjaeYoo** [comment]
Official Loom SDK telegram channel : https://t.me/LoomSDK

They will help you if you send them your deposit txid in ethereum :)

**Comment by YoonjaeYoo** [comment]
It has been resolved in v1.1.4
